patternlab/styleguide/source/assets/js/modules/mainNavMixed.js

```diff
@@ -1,293 +1,27 @@
-      itemButton.setAttribute('aria-label', 'hide menu');
-      setTimeout(function timeoutFunction() {
-        item.removeAttribute('style');
-      }, 700);
-    }
-
-
-    if (subMenu.classList.contains('is-closed')) {
-
-      /** Show the subMenu. */
-
-      subMenu.classList.remove('is-closed');
-      subMenu.style.height = "auto";
-
-      /** Get the computed height of the subMenu. */
-      var height = subMenu.clientHeight + "px";
-
-
-      /** Set the height of the submenu as 0px, */
-      /** so we can trigger the slide down animation. */
-      subMenu.style.height = "0";
-
-      setTimeout(function timeoutFunction() {
-        subMenu.style.height = height;
-        subItems.style.opacity = "1";
-      }, 50);
-
-      /** Slide up. */
-    } else {
-      subMenu.style.height = "0";
-      subItems.style.opacity = "0";
-
-      setTimeout(function timeoutFunction() {
-        subMenu.classList.add('is-closed');
-
-      }, 500);
-    }
-  });
-
-  itemButton.addEventListener('keydown', function (e) {
-
-    if (e.code == 'ArrowDown') {
-      let first = subItems.getElementsByTagName("li")[0];
-      first.querySelector('.js-main-nav-hamburger__link').focus()
-    }
-
-    if (e.code == 'Escape' || e.which == '27') {
-      if (item.classList.contains('submenu-open')) {
-        subItems.style.opacity = "0";
-        subMenu.style.height = "0";
-        itemButton.parentElement.classList.remove('submenu-open');
-        itemButton.setAttribute('aria-expanded', 'false');
-        itemButton.setAttribute('aria-label', 'show menu');
-
-        setTimeout(function timeoutFunction() {
-          subMenu.classList.add('is-closed');
-        }, 500);
-        
-      }
-      else {
-        closeMenu();
-      }
-    }
-
-    if (e.shiftKey && e.code == 'Tab') {
-      subItems.style.opacity = "0";
-      subMenu.style.height = "0";
-      item.classList.remove('submenu-open');
-      itemButton.setAttribute('aria-expanded', 'false');
-      itemButton.setAttribute('aria-label', 'show menu');
-
-      setTimeout(function timeoutFunction() {
-        subMenu.classList.add('is-closed');
-      }, 500);
-    }
-
-    if (width > 840 && e.key == 'Tab') {
-      if (!itemButton.parentElement.nextElementSibling 
-      && !itemButton.parentElement.classList.contains('submenu-open')) {
-        closeMenu();
-      }
+let width = document.documentElement.clientWidth;
+const menuButton = document.querySelector(".js-header-menu-button");
+// let menuButtonText = document.querySelector(".js-header__menu-text");
+// let buttonLabel;
+// const hamburgerMenuContainer = document.querySelector(".ma__header__hamburger__nav-container");
+// const feedbackButton = document.querySelector(".ma__fixed-feedback-button");
+// let menuItems = document.querySelectorAll(".js-main-nav-hamburger-toggle");
+const menuOverlay = document.querySelector(".menu-overlay");
+// let utilNavWide = document.querySelector(".js-utility-nav--wide");
```

Comment:
Let's remove these comments if no longer needed

Reply:
The comment has been removed.